dm/worker/subtask.go

```diff
@@ -75,7 +75,7 @@ func createRealUnits(cfg *config.SubTaskConfig, etcdClient *clientv3.Client, wor
 
 func newLoadUnit(cfg *config.SubTaskConfig, etcdClient *clientv3.Client, workerName string) unit.Unit {
 	// tidb-lightning doesn't support column mapping currently
-	if cfg.ImportMode == config.LoadModeLoader || cfg.OnDuplicate == config.OnDuplicateError || len(cfg.ColumnMappingRules) > 0 {
+	if cfg.ImportMode == config.LoadModeLoader || len(cfg.ColumnMappingRules) > 0 {
```

Comment:
I remember we have a `NeedUseLightning` function. Can we reuse it here?

Reply:
https://github.com/pingcap/tiflow/pull/7714/commits/0121d7e857c66aeaac22dcb0a04ecbb6223f264d